### ISSUE TITLE
Error on export/import when instance name contains underscore

### ISSUE THREAD

**Issue by tbongers-cat** [conceptsandtraining/doil#281]
`doil pack:export ilias7_test`

results in error

```
In ExportCommand.php line 134:                                                             
  Invalid characters! Only letters and numbers are allowed!
```

As underscore is an accepted character in instance names, it should be accepted in pack:export as well.

**Comment by tbongers-cat** [comment]
The same happens on `doil pack:import `when the instance name contains an underscore.

**Comment by daniwe4** [comment]
https://github.com/conceptsandtraining/doil/pull/284

**Comment by tbongers-cat** [comment]
Fixed in #284
